### PR TITLE
Restricted home and destination blockchains

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: src/Contexts/NetworkManagerContext.tsx [hunk 1]
```diff
@@ -131,22 +132,30 @@ const NetworkManagerProvider = ({ children }: INetworkManagerProviderProps) => {
       }
       const chain = homeChains.find((c) => c.chainId === chainId);
 
+      const fetchDestinationChainId = (homeChainId: number) => {
```
Comment (AndreiNavoichyk) [inline src/Contexts/NetworkManagerContext.tsx]: Few can be returned. Rename to plural.

File: src/Contexts/NetworkManagerContext.tsx [hunk 2]
```diff
@@ -131,22 +132,30 @@ const NetworkManagerProvider = ({ children }: INetworkManagerProviderProps) => {
...
+        switch (homeChainId) {
+          case blockchainChainIds.POLYGON:
+          case blockchainChainIds.ETHEREUM:
+            return [blockchainChainIds.CERE];
+          case blockchainChainIds.CERE:
+            return [blockchainChainIds.POLYGON];
```
Comment (AndreiNavoichyk) [inline src/Contexts/NetworkManagerContext.tsx:141]: Return for default empty array.

File: src/Contexts/NetworkManagerContext.tsx [hunk 3]
```diff
@@ -131,22 +132,30 @@ const NetworkManagerProvider = ({ children }: INetworkManagerProviderProps) => {
...
+
       if (chain) {
+        const destinationChainId = fetchDestinationChainId(chain.chainId);
         setHomeChainConfig(chain);
         setDestinationChains(
-          chainbridgeConfig.chains.filter(
-            (bridgeConfig: BridgeConfig) =>
-              bridgeConfig.chainId !== chain.chainId
+          chainbridgeConfig.chains.filter((bridgeConfig: BridgeConfig) =>
+            destinationChainId?.includes(bridgeConfig.chainId)
```
Comment (AndreiNavoichyk) [inline src/Contexts/NetworkManagerContext.tsx]: Check for null is extra.

File: src/Contexts/NetworkManagerContext.tsx (hunk 1 above)
Comment (karthikeyan-cere) [reply src/Contexts/NetworkManagerContext.tsx]: Renamed

File: src/Contexts/NetworkManagerContext.tsx (hunk 2 above)
Comment (karthikeyan-cere) [reply src/Contexts/NetworkManagerContext.tsx:141]: Added

File: src/Contexts/NetworkManagerContext.tsx (hunk 3 above)
Comment (karthikeyan-cere) [reply src/Contexts/NetworkManagerContext.tsx]: It is added while applying the eslint.

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**